packages/@aws-cdk/cloud-assembly-schema/schema/cloud-assembly.version.json

```diff
@@ -1 +1 @@
-{"version":"16.0.0"}
\ No newline at end of file
+{"version":"17.0.0"}
```

Comment:
Curious - do we bump the major version of the schema with every change? Is this change a breaking change? (If this is a silly question, please explain why. 😄 I don't fully understand the contract here. Is the API only used by us?)

Reply:
We do bump the MV of the schema with every change. Every change is indeed potentially breaking (at any point the framework could be requesting services from the CLI that the CLI may not know how to provide--i.e., CLI needs an update to match).

It's a bit onerous to do this though, and you're right that in this particular case it's probably not necessary. This was just forced upon me by the tools we're using. Will revert the version.